internal/controller/sriov_controller.go

```diff
@@ -164,6 +184,41 @@ func (r *SRIOVReconciler) generateOVSNetwork(rail *config.Rail, mtu uint) *sriov
 	return network
 }
 
+// generateSriovNetworkPoolConfig generates a SriovNetworkPoolConfig object
+func (r *SRIOVReconciler) generateSriovNetworkPoolConfig() *sriovnetworkv1.SriovNetworkPoolConfig {
+	nodeSelector := metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			sriovNodePolicyNodeSelector: "",
+		},
+	}
+
+	ovsHardwareOffloadConfig := sriovnetworkv1.OvsHardwareOffloadConfig{
+		OvsConfig: map[string]string{
+			"hw-offload": "true",
+			"dpdk-init":  "true",
+			"doca-init":  "true",
```

Comment:
dpdk is not required.
we have other setting that are needed:
```
ovs-vsctl set Open_vSwitch . other_config:doca-init=true
ovs-vsctl set Open_vSwitch . other_config:hw-offload=true
ovs-vsctl set Open_vSwitch . other_config:doca-eswitch-max=8
ovs-vsctl set Open_vSwitch . other_config:hw-offload-ct-size=0
```

Reply:
done